__tests__/app.test.js

```diff
@@ -43,4 +43,40 @@ describe("GET /api", () => {
             })
         })
     })
+    describe("/articles", () => {
+        describe("/:article_id", () => {
+            test("200: Should retrieve an article by id", () => {
+                return request(app)
+                .get('/api/articles/1')
+                .expect(200)
+                .then((res) => {
```

Comment:
you could destructure {body} here to save some time

Reply:
This has been refactored.